epp-client/src/extensions/namestore.rs

```diff
@@ -71,8 +73,19 @@ impl NameStore {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, ElementName)]
+pub enum ExtensionResponse {
```

Comment:
Hmm, I think I would make a separate type here that is only used for `domain::info`, so that the other requests are not affected by this oddity.

Reply:
Yeah that seems reasonable, I'll play with that.